.circleci/config.yml

```diff
@@ -25,16 +26,19 @@ jobs:
 
   code-check:
     docker: 
-      - image: golang:1.10.4
+      - image: pouchcontainer/pouchlinter:v0.1
     working_directory: /go/src/github.com/alibaba/pouch
     steps:
       - checkout
       - run: 
-          name: install gometalinter linters tools
-          command: hack/install/install_ci_related.sh
+          name: validate swagger.yml
+          command: |
+            swagger validate "/go/src/github.com/alibaba/pouch/apis/swagger.yml"
+            
       - run: 
-          name: use check target in Makefile to check code
-          command: make check
+          name: validate go code with gometalinter
+          command: |
+            gometalinter --disable-all --skip vendor -E gofmt -E goimports -E golint -E ineffassign -E misspell -E vet -E megacheck ./...
```

Comment:
one more question, could we just use `make check` here, since the image provides the binary?

Reply:
I think both are OK, while if we use `make check`, the calling path is a little bit long. For the test part and ci part, in my opinion, **simplicity** is the most important thing.